src/app/money/import/mappings.ts

```diff
@@ -123,6 +123,11 @@ export const MAPPINGS_BY_FORMAT: { [K in ImportFileFormat]: FormatMapping } = {
     .addMapping("amount", "Betrag (EUR)", row => parseAmount(row, ",", "."))
     .addMapping("bookingText", "Kategorie")
     .build(),
+  'genENG': new FormatMappingBuilder<Generic_Eng>()
+    .addMapping("date", "Date", parseDate)
+    .addMapping("reason", "Description")
+    .addMapping("amount", "Amount", parseAmount)
+    .build(),
```

Comment:
Please also map the remaining fields (who, whoIdentifier and bookingText). They will be marked as required but if someone doesn't want to use them, they can just leave it empty.